Loading python/3.7.3
  Loading requirement: tcl/8.6.8 gcc/8.3.0
Namespace(batch_size=512, cy_div=2, cy_scale=2, dataloader_num_workers=4, dataset_path_test='data.nosync/speech_commands_test_set_v0.02', dataset_path_train='data.nosync/speech_commands_v0.02', epochs=20000, global_beta=1.5, hidden=256, hop_length=320, hp_bw=False, init_factor=2, learning_rate=0.0005, lr_divide=10000, n_mfcc=40, noise_injection=0.1, quant_actMVM=6, quant_actNM=8, quant_inp=4, quant_w=None, sample_rate=16000, std_scale=2, testing_percentage=10, validation_percentage=10, validation_size=4000, win_length=150, word_list=['yes', 'no', 'up', 'down', 'left', 'right', 'on', 'off', 'stop', 'go', 'unknown', 'silence'])
eecd2788-bd05-45bf-a11b-bac6df648b0d
Start Training:
Epoch     Train Loss  Train Acc  Vali. Acc  Time (s)
Traceback (most recent call last):
  File "KWS_LSTM.py", line 380, in <module>
    output = model(x_data)
  File "/afs/crc.nd.edu/user/c/cschaef6/.local/lib/python3.7/site-packages/torch/nn/modules/module.py", line 550, in __call__
    result = self.forward(*input, **kwargs)
  File "KWS_LSTM.py", line 300, in forward
    lstm_out, self.hidden_state = self.lstmL(inputs, self.hidden_state)
  File "/afs/crc.nd.edu/user/c/cschaef6/.local/lib/python3.7/site-packages/torch/nn/modules/module.py", line 550, in __call__
    result = self.forward(*input, **kwargs)
  File "KWS_LSTM.py", line 247, in forward
    out, state = self.cell(inputs[i], state)
  File "/afs/crc.nd.edu/user/c/cschaef6/.local/lib/python3.7/site-packages/torch/nn/modules/module.py", line 550, in __call__
    result = self.forward(*input, **kwargs)
  File "KWS_LSTM.py", line 207, in forward
    gates = (CustomMM.apply(quant_pass(input, self.ib, True), self.weight_ih.t(), input, self.noise_level, self.hp_bw) + self.bias_ih + noise_bias_ih + CustomMM.apply(quant_pass(hx, self.ib, True), self.weight_hh.t(), hp_hx, self.noise_level, self.hp_bw) + self.bias_hh + noise_bias_hh)
  File "KWS_LSTM.py", line 164, in forward
    output = input.mm(weight + noise_w)
RuntimeError: CUDA out of memory. Tried to allocate 16.00 MiB (GPU 0; 11.78 GiB total capacity; 10.66 GiB already allocated; 15.38 MiB free; 10.70 GiB reserved in total by PyTorch)
